### PR TITLE
[#177003714] Add GetPMWallet func with mocked response

### PR DESCRIPTION
**Pull request by BurnedMarshal**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "noImplicitAny": true,
     "declaration": false,
     "sourceMap": true,
-    "lib": [],
+    "lib": ["dom"],
```
Comment (AleDore) [inline tsconfig.json:14]: What's the purpose of this lib?

Comment (BurnedMarshal) [reply tsconfig.json:14]: Without this, the transpiler returns errors on autogenerated code for the client, below an example.
```
generated/api/client.ts:99:20 - error TS2304: Cannot find name 'fetch'.

99   fetchApi: typeof fetch;
                      ~~~~~

node_modules/@pagopa/ts-commons/lib/requests.d.ts:50:53 - error TS2304: Cannot find name 'Response'.

50 export declare type ResponseDecoder<R> = (response: Response) => Promise<t.Validation<R> | undefined>;
                                                       ~~~~~~~~

node_modules/@pagopa/ts-commons/lib/requests.d.ts:83:44 - error TS2304: Cannot find name 'FormData'.

83     readonly body: (params: P) => string | FormData;
```

**Comment by BurnedMarshal** [comment]
@AleDore I've updated and fixed the dependencies inside the `package.json` in cd8d288

**Review by AleDore — APPROVED** [review]
LGTM

**Pull request merged**